Makefile

```
@@ -171,6 +171,19 @@ install-mock-server: #? install the mock server in editable mode.
 install-mock-amdsmi: #? install the fake amdsmi bindings for testing the AMD telemetry path.
 	$(activate_venv) && uv pip install -e "tests/aiperf_mock_amdsmi[dev]"
 
+install-hooks: #? write the no-stash pre-commit hook and pre-install hook environments.
```

checkmake (0.3.2)

[warning] Target body for "install-hooks" exceeds allowed length of 5 lines (11).

(maxbodylength)